CHANGES.rst

```diff
@@ -73,6 +73,8 @@ Bug Fixes
 
 * Loading valid data no longer emits JSON serialization warnings. [#2011]
 
+* Fixed linking issue preventing smoothed spectrum from showing in Specviz2D. [#2023]
```

Comment:
Docs have it as capital D: https://jdaviz.readthedocs.io/en/latest/specviz2d/index.html

Reply:
Oh, weird. I thought I remember we agreed to rebrand so that only first letter is capitalized (when we dropped the capital V). Also the class name has lowercase "d".

https://github.com/spacetelescope/jdaviz/blob/71a514c1cf7741874d2770f34c5d0ff9045038a7/jdaviz/configs/specviz2d/helper.py#L12

Now I am confused. But I guess this discussion should not block merge. I already approved. Thanks for your patience!